se/se_epub_lint.py

```diff
@@ -591,6 +591,22 @@ def _recursive_helper(rules: cssutils.css.cssstylesheet.CSSStyleSheet, local_css
 
 	return (local_css_rules, duplicate_selectors)
 
+def files_not_in_spine(self) -> set:
+	"""
+	Check the spine against the actual files.
+
+	INPUTS
+	None
+
+	OUTPITS
+	Set of files not in the spine (typically an empty set)
+	"""
+
+	xhtml_files = set(self.path.glob("src/epub/**/*.xhtml"))
+	spine_files = set(self.spine_file_paths)
+	spine_files.add(self.toc_path)
```

Comment:
We need to use `self.content_path` instead of hard coding `src/epub/` because different ebooks may have different structures. There might also be the case where the ToC is the same folder as the text, so we'd add it twice. I think we first generate a list of all possible xhtml files, THEN make them a set to remove duplicates - not make them a set first, and then add more items later.

Reply:
OK, done.